### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by VladyslavKolisnyk**
DEMO LINK https://VladyslavKolisnyk.github.io/layout_catalog
TEST REPORT LINK https://VladyslavKolisnyk.github.io/layout_catalog/report/html_report

### CODE REVIEW

**Review by slavanosatskyi — CHANGES_REQUESTED** [review]
please, fix demo and test report links. They lead to 404 
https://VladyslavKolisnyk.github.io/layout_catalog/

File: src/styles/blocks/cards.scss [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-dark-blue;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-style: normal;
+    font-weight: 400;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/cards.scss:27]: remove redundant default properties

File: src/styles/blocks/cards.scss [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    width: 166px;
+    height: 40px;
+    text-align: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    background: $color-blue;
+    border-radius: 5px;
+    color: #fff;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/cards.scss:78]: move colors into a constants

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
1. layout shouldn't be broken if I add some more cards
image

**Review by OleziO — APPROVED** [review]

**Pull request closed without merging**